src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```diff
@@ -32,14 +32,16 @@ namespace DurableTask.AzureStorage
     using DurableTask.Core.History;
     using Microsoft.WindowsAzure.Storage;
     using Newtonsoft.Json;
+    using Core.Query;
```

Comment:
nit: let's make this explicit: 
```suggestion
    using DurableTask.Core.Query;
```
We should also move it up with the other DurableTask.Core.* using statements in alphabetical order. The fact that you can do just `Core.Query` is one of the odd behaviors in C# when `using` statements go inside the `namespace` declaration. I find it quite confusing...